src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -189,7 +234,12 @@ public async Task<Stream> GetFileAsync(long version, Partition partition, FilePr
         // We should either remove fo-dicom parsing for transcoding or make SDK change to support Length property on RetrievableStream
         //Response<BlobDownloadStreamingResult> result = await blobClient.DownloadStreamingAsync(range: default, conditions: null, rangeGetContentHash: false, cancellationToken);
         //stream = result.Value.Content;
-        return await ExecuteAsync(async () => await blobClient.OpenReadAsync(blobOpenReadOptions, cancellationToken));
+        return await ExecuteAsync(async () =>
+        {
+            Stream stream = await blobClient.OpenReadAsync(blobOpenReadOptions, cancellationToken);
+            EmitTelemetry("GetFileAsync", OperationType.Output, stream.Length);
```

Comment:
I think you're ok with this `Length` too: https://github.com/Azure/azure-sdk-for-net/blob/fba9f87d8e6c44ad213c7f6b9844d32d86ea8edb/sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs#L3167